src/member/templatetags/member_extras.py

```diff
@@ -18,10 +19,17 @@ def dish_size(context, day):
 @register.filter
 def readable_prefs(value):
     if value is None:
-        return ''
+        return _("Not properly configured.")
```

Comment:
If the day is selected, but no defaults (and it is possible as it is separated in two fields), the system default should be used: 1 regular main dish, 1 dessert.